migtests/tests/resumption/pg/resumption/config.yaml

```diff
@@ -0,0 +1,57 @@
+# Additional Flags
+# Uncomment the below to add any additional flags to the command
+
+# additional_flags:
+#   --delimiter: "\\t"
+
+# Import Type
+import_type: offline
+
+# Row Count Validation
+row_count:
+  Case_Sensitive_Table: 5000000
+  case: 5000000
+  Table: 5000000
+  schema2.Case_Sensitive_Table: 5000000
+  schema2.case: 5000000
+  schema2.Table: 5000000
+  public.boston: 2500000
```

Comment:
where is the code that generates data for all these other tables boston/cust/emp/etc?

Reply:
Those are being done via the partitions test schema / data